corehq/apps/analytics/static/analytics/js/kissmetrics.js

```diff
@@ -0,0 +1,152 @@
+/* globals _, _kmq */
+
+var _kmq = _kmq || [];
```

Comment:
I know `_kmq` comes from kiss, but could it get a reasonable name once it's in our code? `KissMetricsQueue`? Naming is one of the things that's irritating about working with kiss.

Reply:
the global `_kmq` variable is what kissmetrics interacts with. I tried looking for a way to rename it and let kissmetrics know with no luck. Outside of this analytics library we shouldn't *ever* be calling `_kmq` directly anywhere on HQ. `_kmq.push` should only ever be seen in `analytics/js/kissmetrics.js`

Reply:
I don't understand - couldn't this line be `var kissMetricsQueue = _kmq || [];`? If it can't, yeah, at least it's localized.